google/cloud/extended_operations_pb2.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
```

Comment:
I believe we need to add these files into 
https://github.com/googleapis/python-api-common-protos/tree/main/google

We have done simiilar procedure in other langauges already, for example:
Java: https://github.com/googleapis/java-common-protos/tree/main/proto-google-common-protos/src/main/java/com/google/cloud

Ruby: https://github.com/googleapis/common-protos-ruby/tree/main/google-cloud-common/lib/google/cloud

Once we do that, I believe we can remove most of these files from this PR.

Reply:
https://github.com/googleapis/python-cloud-common/tree/main/google/cloud/common/types was created for `google/cloud/common_resources.proto`, so I think it will be a more appropriate repo/package for `extended_operations.proto`.

Can the BUILD.bazel https://github.com/googleapis/googleapis/blob/master/google/cloud/BUILD.bazel be updated to generate this proto as well?

Another note, we've switched to generating proto-plus types by default instead of _pb2.py, will that be a problem for the generator or generated library code? @software-dov

Reply:
https://github.com/googleapis/googleapis/blob/master/google/cloud/BUILD.bazel#L51 already has a python target for extended_operations. But that one will create default stubs. If you want generate proto-plus ones, then it is still straightforward, but woluld require changing that target to py_gapic_library and py_gapic_assembly targets instead. I'm happy to assist with that.

Reply:
The `extended_operations_pb2.py` file is just used by the generator; I don't believe generating proto-plus types will work because proto-plus can't handle message extensions.

Reply:
@vam-google The other files added to the PR are necessary for the fragment tests. The only file that can and should be moved is `extended_operations_pb2.py`.